init.ps1

```diff
@@ -52,14 +47,11 @@ UpdatePackage schema test_schema_library "schema"
 UpdatePackage tools schema_compiler-x86_64-win32 "$SdkPath/.schema_compiler"
 UpdatePackage tools schema_compiler-x86_64-macos "$SdkPath/.schema_compiler"
 
-UpdateSpot spot-win64 "$SdkPath/.spot/spot.exe"
-UpdateSpot spot-macos "$SdkPath/.spot/spot"
-
 #Update Mobile SDK
 UpdatePackage worker_sdk c-static-arm-clang-ios "$SdkMobilePath/Plugins/Improbable/Core/iOS/arm"
 UpdatePackage worker_sdk c-static-x86_64-clang-ios "$SdkMobilePath/Plugins/Improbable/Core/iOS/x86_64"
 
-UpdatePackage worker_sdk c-dynamic-arm64v8a-clang_ndk21-android "$SdkMobilePath/Plugins/Improbable/Core/Android/arm64"
-UpdatePackage worker_sdk c-dynamic-armv7a-clang_ndk21-android "$SdkMobilePath/Plugins/Improbable/Core/Android/armv7"
+UpdatePackage worker_sdk c-dynamic-arm64v8a-clang_ndk21d-android "$SdkMobilePath/Plugins/Improbable/Core/Android/arm64"
```

Comment:
Does this change mean the debug packages are used? Is this intended?

Reply:
Nope, NDK 21d is the LTS version of NDK 21 (https://developer.android.com/ndk/downloads).

I'm unclear why the revisions are named as such.